blueocean-dashboard/coverage/clover.xml

```diff
@@ -0,0 +1,4452 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files were checked in accidentally. You'll want to delete everything in `blueocean-dashboard` `coverage` and `reports` if they exist. If you merge latest master and re-run build, these files will be instead placed in `./target/jest-coverage` and `./target-jest-reports` respectively which will already be ignored.